packages/canary-web-components/src/components/ic-date-input/test/basic/__snapshots__/ic-date-input.spec.tsx.snap

```diff
@@ -497,6 +497,54 @@ exports[`ic-date-input component should render as required 1`] = `
 </ic-date-input>
 `;
 
+exports[`ic-date-input component should render as required 2`] = `
```

Comment:
Nothing in this snapshot suggests that an asterisk has been added. Worth updating the required test to assert an asterisk exists if not already done.

Reply:
this one is checking that the asterisk has been removed i.e. the span is not in the snapshot